ports/esp32/machine_touchpad.c

```diff
@@ -106,19 +110,32 @@ STATIC mp_obj_t mtp_make_new(const mp_obj_type_t *type, size_t n_args, size_t n_
 
 STATIC mp_obj_t mtp_config(mp_obj_t self_in, mp_obj_t value_in) {
     mtp_obj_t *self = self_in;
+    #if CONFIG_IDF_TARGET_ESP32
     uint16_t value = mp_obj_get_int(value_in);
     esp_err_t err = touch_pad_config(self->touchpad_id, value);
     if (err == ESP_OK) {
         return mp_const_none;
     }
+    #elif CONFIG_IDF_TARGET_ESP32S2 || CONFIG_IDF_TARGET_ESP32S3
+    uint32_t value;
+    esp_err_t err = touch_pad_read_raw_data(self->touchpad_id, &value);
```

Comment:
This should configure the touchpad, not read the raw data.

Reply:
Corrected and force-pushed.